tests/test_colon.py

```diff
@@ -29,14 +29,10 @@ def test_colon1(self):
         self.assertEqual(parameterSetNames, ["Default", "Cattle 1", "Human 1", "Human 2", "Human 3", "Mouse 1",
                                              "Mouse 2", "Pig 1"])
 
-        centralPathDefaultScaffoldPackages = {
-            'Test line': ScaffoldPackage(MeshType_1d_path1, {
+        parameterSetStructureStrings = {
```

Comment:
Here and in test small intestine.
Can you rename this and similar variables which aren't structure strings.
Similarly where used below
centralPathOption = parameterSetStructureStrings['Test line']
and
'Central path': copy.deepcopy(centralPathOption)
(surely the above should now be 'Network layout'?)
Rather than copy.copy, best is to construct the ScaffoldPackage in the above line with the dictionary looked up for 'Test line'.
e.g. 
'Network layout': ScaffoldPackage(MeshType_1d_network_layout1, networkLayoutScaffoldPackageDct['Test line'])
There is also a bit of test code with variables centralPath which should be networkLayout.